rollup.config.js

```diff
@@ -1,16 +1,20 @@
-import pkg from './package.json';
-import commonjs from 'rollup-plugin-commonjs';
-import nodeBuiltins from 'rollup-plugin-node-builtins';
-import nodeResolve from 'rollup-plugin-node-resolve';
-import babel from 'rollup-plugin-babel';
+'use strict';
+
+const pkg = require('./package.json');
+const commonjs = require('rollup-plugin-commonjs');
+const nodeGlobals = require('rollup-plugin-node-globals');
+const nodeBuiltins = require('rollup-plugin-node-builtins');
+const nodeResolve = require('rollup-plugin-node-resolve');
+const babel = require('rollup-plugin-babel');
 
 const input = 'index.js';
 const plugins = [
   nodeResolve(),
   commonjs(),
   nodeBuiltins(),
+  nodeGlobals(),
```

Comment:
It seems like this shouldn't be added back in. This is going to polyfill things that don't need to be there based on latest changes.